app.js

```diff
@@ -0,0 +1,33 @@
+console.log("Started");
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const hbs = require('express-handlebars');
+const products = require('./routes/products');
+const articles = require('./routes/articles');
+
+const port = process.env.PORT || 2222;
+const app = express();
+
+app.engine('.hbs', hbs({
+  defaultLayout : 'main',
+  extname : '.hbs'}));
+
+app.set('view engine', '.hbs');
+app.use(bodyParser.urlencoded({ "extended" : true }));
+app.use(express.static('public'));
```

Comment:
express.static middleware should be the first one opened up so that request to static files don't hit all other middleware.